fal/controllers/anime_stats.py

```diff
@@ -59,14 +75,30 @@ def get_anime_stats_from_jikan(anime: Anime) -> Dict[str, Union[int, float]]:
     general_anime_info = jikan.anime(anime.id)
     jikan_anime_stats = jikan.anime(anime.id, extension='stats')
 
-    return {
-        'watching': jikan_anime_stats['watching'],
-        'completed': jikan_anime_stats['completed'],
-        'dropped': jikan_anime_stats['dropped'],
-        'score': general_anime_info['score'],
-        'favorites': general_anime_info['favorites'],
-        'forum_posts': get_forum_posts(anime)
-    }
+    return AnimeStats(
+        watching = jikan_anime_stats['watching'],
+        completed = jikan_anime_stats['completed'],
+        dropped = jikan_anime_stats['dropped'],
+        score = general_anime_info['score'],
+        favorites = general_anime_info['favorites'],
+        forum_posts = get_forum_posts(anime)
+    )
+
+def calculate_anime_weekly_points(stat_data: AnimeStats) -> int:
+    '''
+    Calculate the points for the week for the anime based on the stats
+    '''
+    points = 0
+    points += (stat_data.watching
+             + stat_data.completed
+             + config.getint('scoring.dropped', str(stat_data.week), fallback=0) * stat_data.dropped
+             + int(config.getint('scoring.anime_score', str(stat_data.week), fallback=0) * stat_data.score)             + config.getint('scoring.favorite', str(stat_data.week), fallback=0) * stat_data.favorites
```

Comment:
Could we move the expression with `scoring.favorite` to the next line so only 1 expression is on each line?

Reply:
Eh, how'd that happen? Will fix